config/database.yml

```diff
@@ -14,5 +14,7 @@ test:
 production:
   <<: *default
   database: intercity
-  host: <%= ENV["DB_SOCKET"] %>
+  host: db
```

Comment:
Shall we perhaps use ENV vars here too? That allows for running the app with external or offloaded databases too. For example, at Firmhouse I would offload the database to an encrypted-at-rest Database service that's maintained by a 3rd party.